packages/neuron-ui/src/widgets/Tabs/index.tsx

```diff
@@ -37,8 +42,7 @@ const DefaultVariant = <T extends any = {}>({
             tabIndex={0}
             onKeyDown={onEnter(() => onTabChange(tab.id))}
             onClick={() => onTabChange(tab.id)}
-            className={tabsColumnClassName}
-            data-active={selectedTab.id === tab.id}
+            className={clsx(tabsColumnClassName, selectedTab.id === tab.id && activeColumnClassName)}
```

Comment:
This requires confirming what the default style looks like, and currently, I'm not entirely sure about the default style of the tabs.
Replacing `data-active` with `className` was done to avoid implicit dependencies from external components on the internal implementation details of this component, thus preventing coupling.